package.json

```diff
@@ -47,6 +47,7 @@
     "moment-timezone": "^0.5.23",
     "mutant": "github:mmckegg/mutant#intersection-binding-viewport",
     "mutant-pull-reduce": "^1.1.0",
+    "node-abi": "^2.7.1",
```

Comment:
Is this used somewhere?

Reply:
This is depended on transitively, I added this minimum dependency to force the upgrade. I suppose it could be removed now that package-lock.json has updated with the higher version. (The earlier version doesn't support Electron 4.)